### DISCUSSION TITLE
Conflicting Command name with another plugin

### DISCUSSION THREAD

**Discussion by mhartington** [scalameta/nvim-metals#123] [General]
Hi there! I am the author of Formatter.nvim, a formatting plugin for neovim. We had an issue reported the repo and long story short, both plugins have a command called `Format`. 

Since most other command in this plugin are prefixed with "Metals-", would you be up for prefixing the Format command as well?

**Comment by ckipp01** [comment]
Hello! Totally. Honestly, I'm just going to remove it, not preface it. `Format` in this context is literally just calling `vim.lsp.buf.formatting()`, and I had this in here since I first started the plugin and was using it to format, and then totally forgot about it.

**Reply by ckipp01** [reply]
Fixed in https://github.com/scalameta/nvim-metals/pull/124
